ADQL.tex

```diff
@@ -116,7 +113,7 @@ \subsection{Role within the VO architecture}
 IVOA architecture \VOArch{}.
 
 It is the only mandatory query language within the Table Access Protocol
-TAP \citep{2019ivoa.spec.0927D}; it is thus an important element in
+\TAPSpec{} ; it is thus an important element in
```

Comment:
Yes and no. A space is needed before the semicolon in french. So, yes, deliberate because of my french habits, and no, because I forgot no space is needed in english. In brief, it can be removed :-)

Reply:
Done